src/components/Price/Conversion.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useState, ReactElement } from 'react'
+import styles from './Conversion.module.css'
+import classNames from 'classnames/bind'
+import { formatCurrency, isCrypto } from '@coingecko/cryptoformat'
+import { useUserPreferences } from '../../providers/UserPreferences'
+import { usePrices } from '../../providers/Prices'
+import { currency } from '../../../app.config'
+
+const cx = classNames.bind(styles)
+
+export default function Conversion({
+  price,
+  className,
+  hideApproximateSymbol,
+  showTVLLabel
+}: {
+  price: string // expects price in OCEAN, not wei
+  className?: string
+  hideApproximateSymbol?: boolean
+  showTVLLabel?: boolean
```

Comment:
issue(blocking): we can remove the TVL label alltogether, as we don't support pools in the mvg demo